Fixes #28786 - new columns on subscription report

waldirio opened the pull request:
Hey @ares do you have any insight about how to get the organization name/list and then iterate on that? I've already added the column, however, I'm not sure how I could retrieve the org label.

Thank you.

waldirio committed:
new columns on subscription report

ares commented on report_templates/subscriptions.erb:
```diff
@@ -21,6 +21,10 @@ require:
+        'Account number': pool.account_number,
+        'Start date': pool.start_date,
+        'End date': pool.end_date,
+        'Organization': "org_label_here",
```
for this, we need to enable organization in safe mode first, I opened https://github.com/Katello/katello/pull/8532
once merged, you can then use `pool.organization.name`

that also needs updating include in load_pools to avoid SQL inefficiency

ares reviewed (dismissed):
once the macro is available, we'll also need to add `:organization` into load_pools includes (to avoid SQL inefficiency), bump required katello version to 3.16 (or the version it's merged into)

ezr-ondrej commented:
Katello PR is merged, @waldirio could you please use the suggestion above? :slightly_smiling_face:

ares commented:
@waldirio please let me know if you need any help with finishing this one, the comments above are still valid, however easy changes I believe

waldirio committed:
calling the new macro pool.organization.name

waldirio commented:
@ares and @ezr-ondrej done. Thank you!!

ares committed:
Required changes

We need to bump minimal Katello version and inlcude organization to avoid N+1 SQL queries

ares committed:
Merge branch 'develop' into fix/28786

ares commented:
I've rebased this for you, leaving now to @ezr-ondrej since I have my code changes in this PR too. Please squash on merge.

ezr-ondrej reviewed (approved):
Thanks @waldirio and @ares !

waldirio commented:
Thank you @ares and @ezr-ondrej 
I was with no time and struggling with the rebase, but now, everything is good.

wclark++
